tests/testthat/_snaps/step-call-pivot_wider.md

```diff
@@ -3,7 +3,7 @@
     Code
       show_query(step)
     Output
-      setnames(dcast(`_DT5`, formula = "..." ~ x + y, value.var = c("a", 
+      setnames(dcast(`_DT6`, formula = "..." ~ x + y, value.var = c("a", 
```

Comment:
Can you please explicitly set the name in the `lazy_dt()` call to stop this from changing in the future?

Reply:
This is the snapshot that is generated when testing that `pivot_wider.data.table()` redirects to `pivot_wider.dtplyr_step()`. Do you have a preferred name for `lazy_dt()` to use inside of `pivot_wider.data.table()` and future `.data.table` methods?

Reply:
Oooooooh. Leave it as for now then, and at some point in the future we should figure out a better solution.

Reply:
Sounds good - all set for review